docs/core/entity/light.md

```diff
@@ -70,6 +72,19 @@ class MyLightEntity(LightEntity):
         """Turn device on."""
 ```
 
+Note that there's no `color_mode` passed to the `async_turn_on` method, instead only a single color is allowed.
+
+Colors in the service call will be translated before the entity's `async_turn_on` method is called if the light doesn't support the corresponding color mode:
+| Color type   | Translation
+|--------------|-----------------------
+| color_temp | Not translated, will be passed as is even if not supported by the light.
+| hs_color | Will be removed from the service call if not supported and translated to `rgb_color`, `rgbw_color`, `rgbww_color` or `xy_color` if supported by the light.
+| rgb_color | Will be removed from the service call if not supported and translated to `rgbw_color`, `rgbww_color`, `hs_color` or `xy_color` if supported by the light.
+| rgbw_color | Not translated, will be passed as is even if not supported by the light.
+| rgbww_color | Not translated, will be passed as is even if not supported by the light.
+| hs_color | Will be removed from the service call if not supported and translated to `hs_color`, `rgb_color`, `rgbw_color` or `rgbww_color` if supported by the light.
```

Comment:
`hs_color` won't be translated to `hs_color`, right?

Reply:
And also:
![image](https://user-images.githubusercontent.com/14281572/116551110-dfcbdc80-a8f7-11eb-9042-949aee5f06bd.png)